### ISSUE TITLE
Code Splitting in Component Causes SyntaxError

### ISSUE THREAD

**Issue by steriley** [vue-styleguidist/vue-docgen-api#50]
I have ran into the following issue when using webpacks Code Splitting feature:

The component uses the feature to load a large carousel library when it is required. However this is causing errors when serving / building:

```
SyntaxError: Carousel.vue: Unexpected token (25:23)
Warning: Cannot parse Carousel.vue:
    Error: SyntaxError: Carousel.vue: Unexpected token (25:23)
```
Carousel.vue line 25 consists of:
`const Carousel = () => import(/* webpackChunkName: "carousel" */ './LargeLibrary.vue');`

I have tried removing the magic comments in the import, but it appears to be the import statement that is tripping up the build process.

The styleguide is generated but this particular Component is obviously missing from the output.

Great project by the way, finding it amazing to use!

**Comment by steriley** [comment]
My bad, this appears to be a duplicate, I have just come across this being reported already by the look of it in #25

**Comment by elevatebart** [comment]
Hello @steriley  ! Can you please install 3.1.0 or vue-styleguidist 3 and tell me if you still have th issue ?

**Comment by steriley** [comment]
@elevatebart I can confirm this is working as expected now after upgrading packages. Thanks!

**Comment by elevatebart** [comment]
Thank you
